src/install.jl

```diff
@@ -131,11 +157,9 @@ function enable_mkl_startup(libmkl_rt)
     end
 
     # Next, build a new system image
-    sysimgpath = PackageCompiler.sysimgbackup_folder("native")
-    if ispath(sysimgpath)
-        rm(sysimgpath, recursive=true)
-    end
-    force_native_image!()
+    PackageCompiler.create_sysimage(:MKL, cpu_target="native", incremental=false, replace_default=true)
```

Comment:
I think you can remove the two lines below and just use the `script="generate_precompile.jl"` option  as described in https://julialang.github.io/PackageCompiler.jl/dev/refs/#References-1.